### DISCUSSION TITLE
HDD

### DISCUSSION THREAD

**Discussion by 0civic0** [Chia-Network/chia-blockchain#6214] [Support]
What happen to my hard drives after the farming operation is over and the hard disks memory are full? Is it possible to empty the hard drives and reuse them, or should I consider buying and adding new hard drives?

**Comment by tiny0little** [comment]
> What happen to my hard drives after the farming operation is over and the hard disks memory are full?

farming will going on, we do not expect it to end. you can continue farming with a drive full of plots

> Is it possible to empty the hard drives and reuse them

you can re-use or sell your drives if you will see a fit for it

> or should I consider buying and adding new hard drives?

you can buy more drive to increase size of your plot space

[Beginners-Guide](https://github.com/Chia-Network/chia-blockchain/wiki/Beginners-Guide)

**Comment by BrandtH22** [comment]
@0civic0 , since this ticket was opened a number of years ago without further comments it seems this issue has been resolved. 
We will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions or comments.

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
